<h1 class="entry-title">Symposium list</h1> </header> <div class="entry-content cf"> <p>Here you can find the list of symposiums selected by the Scientific Committee. The numbering of symposiums is the same as in the registration system for the conference.</p> <h3>Sex and sexes</h3> <div id="wp-accordion-1" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">1. Ecology and the evolution of sex. Lutz Becks, Hanna Koch</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Ecology and the Evolution of Sex</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">We are still missing the answer to one of the important questions in evolutionary biology: Why sex? Despite the high prevalence of sexual reproduction in nature, understanding its evolution and maintenance is still not that straightforward. The reason lays partly in the scarcity of experimental tests and theory that account for ecological dynamics and their direct consequences, i.e. population cycling and thus changes in strength and/or direction of selection. An important step towards solving the mystery of sex is to include ecological dynamics in experimental, theoretical as well as genomic studies. With this proposed symposium, we hope to start a rapid growing discussion on how to further integrate the role of ecology into the field of the evolution of sexual reproduction.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Levi Morran and Tanja Schwander</p> <p></div></div></p> <p><h3 class="wp-tab-title">2. What is new in the study of sex allocation? Bram Kuijper, Sara Magalhaes</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">What is new in the study of sex allocation?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Sex allocation is one of the most quintessential topics in evolutionary biology. Given the vast array of studies since the revolutionary insights of Fisher and Hamilton, it is difficult to conceive of sex allocation as an exciting novel topic. Yet, sex allocation is one of the most prolific areas of research which has being reinventing itself in the last decade, helped by the advent of genomics, the availability of long-term datasets and experimental evolution. For example, during the last 6 years the field has witnessed important discoveries, such as a confirmation of the adaptive basis of condition-dependent sex allocation, experimental evolution of sex ratios in viscous populations and climate driven divergence of sex determining mechanisms to name but a few. Surprisingly, however, no symposium on sex allocation was offered in the previous three ESEB conferences.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">We aim to fill this gap by organizing a seminar which stars two upcoming leaders of the field, Lukas Schärer (UBasel) who uses genomics approaches and experimental evolution to study sex allocation in simultaneous hermaphrodites and Lisa Schwanz (UNewSouthWhales) who has done groundbreaking work on the ecological underpinnings of sex allocation in mammals. We are therefore certain that this symposium will be of great interest to the audience of ESEB.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Lukas Scharer and Lisa Schwanz</p> <p></div></div></p> <p><h3 class="wp-tab-title">3. The evolution of sex chromosomes. Susana Coelho, Nicolas Perrin</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">The Evolution of Sex Chromosomes</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The evolution of sex determination is a major question in evolutionary biology. Until recently, however, our views on sex determination and on the mechanisms driving sex chromosome evolution have been heavily based on data from only a handful of classically studied model organisms (e.g. <em>Mus</em>, <em>Drosophila</em>, <em>Caenorhabditis</em>, <em>Silene</em>), with a focus on the evolutionary consequences of recombination arrest. With the recent advent of the genomic era, this field is now experiencing an empirical renaissance, expanding at an unprecedented pace. Next-generation tools have already led to a flurry of new discoveries. Attention has broadened to non-model organisms, including algae and fungi where sex is determined at the haploid level. Studies on fish, amphibians and reptiles are imposing the view that degeneration is not the ineluctable destiny of sex chromosomes, and that the old dichotomy between genetic and environmental sex determination should be reappraised. Most importantly, the mechanisms of sex determination appear now evolutionarily much more labile than thought just one decade ago. What drives the surprising dynamics of such a fundamental process that, at the end, always leads to the same and simple output, i.e. the production of males and females? With this Symposium, we hope to gather theoreticians and empiricists working on a diversity of systems, and interested in the molecular mechanisms, ultimate causes, and evolutionary consequences of sex chromosome evolution.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Doris Bachtrog and James Umen</p> <p></div></div></p> <p><h3 class="wp-tab-title">4. Evolutionary consequences of sexually antagonistic selection. Brian Hollis, Göran Arnqvist</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolutionary consequences of sexually antagonistic selection</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Males and females share a genome but are often subject to divergent selection. This simple fact results in tension between patterns of male and female adaptation. The aim of this symposium is to highlight phenotypes that experience intralocus sexual conflict as well as their genomic basis, in both laboratory and natural systems. This includes conflicts between the sexes in reproductive strategies, ecological niches, rates of aging, and gene expression, for example. We will also explore the larger significance of sexually antagonistic selection in the maintenance of genetic variation and the evolution of sex chromosomes.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Stephen Chenoweth and William Rice</p> <p></div></div></p> <p><h3 class="wp-tab-title">5. Novel insights in the genetics of sex-specific variation. Elina Immonen, Holger Schielzeth</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Novel insights in the genetics of sex-specific variation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Sex-specific selection is ubiquitous and has resulted in widespread sexual dimorphism in morphology, physiology and behavior. The evolution of sexual dimorphism depends on sex-specific genetic variance despite the fact that females and males share most of the genome. This symposium aims to showcase recent advances in our understanding of the genetics of sex-specific variation and its contribution to sexual dimorphism. Some of the specific areas of interest include identification of genetic basis for sexually dimorphic or sex-specific traits, the role of sex chromosomes in harboring sex-specific variance, the degree to which sex-specific evolution is constrained by intersexual genetic correlations and the link between sexual dimorphism at the molecular and phenotypic levels. We invite researchers working on genetics of sexual dimorphism using quantitative genetic, genomic and transcriptomic approaches to contribute to the symposium.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Daphne Fairbairn and Tim Connallon</p> <p></div></div></p> <p></div> <p>&nbsp;</p> <h3>Social interactions</h3> <div id="wp-accordion-2" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">6. Mating system evolution: unifying theory and test. Greta Bocedi, Francisco Garcia-Gonzalez</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Mating system evolution: unifying theory and test</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Mating systems shape the transmission of alleles into future generations. A central objective in evolutionary ecology is therefore to understand what components of sex-specific selection drive mating system evolution, and how resulting mating systems can in turn shape population ecology, physiology and behaviour and affect the evolutionary dynamics of other life-history traits. Yet the processes that drive the evolution of key mating systems, such as polyandry, self-fertilisation and inbreeding, remain widely debated. This symposium will bring together theoreticians and empiricists working across the plant-animal divide to highlight recent advances in the theory of mating system evolution and empirical test. Particularly, the aim will be to facilitate integration between emerging empirical approaches and increasingly genetically explicit theory, and identify ways in which these approaches can be most constructively combined.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Suzanne Alonzo, Aneil Agrawal</p> <p></div></div></p> <p><h3 class="wp-tab-title">7. Social evolution and sexual conflict. Tommaso Pizzari, Jay Biernaskie</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Social Evolution &amp; Sexual Conflict</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Competition over mates naturally involves social traits. For example, competing males often harm each other and/or the females that they compete over. The study of such sexual conflicts has exploded both theoretically and empirically in the last 20 years. However, most of this work has ignored the fact that natural populations are structured by different degrees of viscosity (e.g., limited dispersal), which means that competition over mates often occurs among individuals that are genetically related (e.g., siblings). Population viscosity plays a fundamental role in the evolution of selfishness and cooperation through its modulation of individuals’ inclusive fitness. Although this is well established in general, there has been a surprising lack of integration of inclusive fitness arguments in the study of sexual conflict. Recent theoretical and empirical work is changing this by showing that kin selection can drastically modulate intrasexual competition and sexual conflict. This symposium aims to bring together theoreticians and empiricists to discuss this emergent line of research in evolutionary biology, foster collaborations and identify new challenges and approaches.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Andy Gardner and Tracey Chapman</p> <p></div></div></p> <p><h3 class="wp-tab-title">8. Cooperation without kinship: from genomes to mutualisms. Arvid Ågren, Kevin Foster</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Cooperation without kinship: from genomes to mutualisms</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Conflict and cooperation can occur at all levels of life. While kin selection is a powerful explanation for some cooperation, there are many cases where relatedness is simply not possible, such as cooperation between species. More generally, these cases are forms of “egalitarian” cooperation and include examples as diverse as genes in a genome, nuclear and organellar genomes within cells, and species in symbiosis and mutualisms. Without relatedness, the key hurdle in egalitarian coalitions is conflict suppression. Conflict suppression mechanisms have been studied in all the key examples of egalitarian coalitions, including the suppression of transposable elements in genomes, and cheaters in mutualisms. However, these systems are rarely discussed together. Our symposium will bring together researchers from across these diverse systems to stimulate interactions and seek general principles underlying the evolution of egalitarian cooperation.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Justin Blumenstiel and Toby Kiers</p> <p></div></div></p> <p><h3 class="wp-tab-title">9. Evolutionary ecology of cooperation: theory and experiment. Dusan Misevic, Sam Brown</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolutionary ecology of cooperation: theory and experiment</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Understanding the ecology and evolution of cooperation remains one of the great challenges in biology. Why do individuals help others at a personal cost? How do patterns of social interaction emerge from behavioral, evolutionary, and ecological processes? What are the selective forces that maintain multi-species cooperative interactions within communities?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Theory has been a vital driver in answering these and other questions, especially when engaged with data. Making the data-theory connections is often complex, as cooperation is studied in the lab, the field and the clinic, at all levels of biological complexity, in natural systems that range from cancerous cells and pathogenic microbes to birds, bees and humans. Robotic and other <em>in silico</em> computational systems have a potential to act as an intermediary between the different approaches to cooperation research. During the symposium we aim to move beyond ineffective debates over kin versus group selection, and focus instead on theory as a problem-solving tool, both fully driven by data and directly informing future experiments.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Testable theoretical predictions and theoretical frameworks based on real-world data are essential for moving the field forward. More than just an overview of theory, simulations, and experiments about cooperation, at this symposium we wish to bring together researchers who aim to make connections between these approaches, refocus our joint efforts, and move the field forward.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Ashleigh Griffin and Jeff Gore</p> <p></div></div></p> <p></div> <p>&nbsp;</p> <h3>Interspecific interactions</h3> <div id="wp-accordion-3" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">10. Adaptation in heterogeneous environments: insights from host-parasite systems. Nicolas Rode, Florence Débarre</h3></p> <p><div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Adaptation in heterogeneous environments: insights from host-parasite systems</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Understanding how species adapt to heterogeneous environments is a major challenge of evolutionary biology. In host-parasite systems, antagonistic coevolution generates highly heterogeneous selective pressures, both in time and space.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Recently, much progress has been made to characterize the process of coevolution in natural (e.g., plant-pathogen, invited speaker ALL) and laboratory (e.g., phage-bacteria, invited speaker AH) populations, showing that the strong and changing selective pressures generated by antagonistic coevolution can play an important role in the dynamics and maintenance of genetic diversity.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Yet, several conceptual challenges and limitations remain. First, transfers of populations in space or time (local adaptation or time-shift experiments), widely used to quantify adaptation, provide a wealth of data on coevolutionary processes, but environmental heterogeneities can complicate their interpretation. Second, experimental work has revealed that the genetics of the interaction may be complex, time- and environment-dependent, and we lack theoretical predictions for the dynamics of coevolution in such scenarios. Third, the extent to which ecological feedbacks (in particular in systems where host and parasite population sizes may vary widely) affect the coevolutionary process remains unclear.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The goal of this symposium is to address these challenges by synthesizing the most recent theoretical and experimental work on host-parasite coevolution.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The idea for the symposium was conceived by a group of three people who are all contributing to its organisation: Nicolas Rode, Florence Débarre and François Blanquart</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Anna-Liisa Laine and Alex Hall</p> <p></div></div></p> <p><h3 class="wp-tab-title">11. Host defence in a parasitized world: selection, evolution and the maintenance of variation. Barbara Tschirren, Lars Råberg</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Host defence in a parasitized world: selection, evolution and the maintenance of variation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Infectious diseases are assumed to be among the most important selective forces in nature. Yet, phenotypic traits involved in host defense against parasites and pathogens, and their underlying genes, often exhibit considerable variation. This symposium aims to integrate studies of selection on host defense at the phenotypic and molecular genetic level across diverse taxa to advance our understanding of how defense mechanisms evolve and how parasite- and pathogen-mediated selection can maintain variation in host defense, both at the level of the individual and the population.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Andrea L. Graham and Brian P. Lazzaro</p> <p></div></div></p> <p><h3 class="wp-tab-title">12. Next-generation phylodynamics. Tanja Stadler, Alexei Drummond</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Next-generation phylodynamics</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The term phylodynamics was coined in the early 2000’s to highlight host-pathogen systems in which evolutionary, epidemiological and immunological processes interact because they proceed on the same time scale. Such interaction is in particular observable for fast-evolving pathogens including RNA viruses and bacteria. When this is the case genomic sequence data sampled from pathogen populations contains a fingerprint of the processes that acted in the past – namely the evolutionary, epidemiological and immunological dynamics. Recent advances in next-generation sequencing technology provide the required data for phylodynamic analysis using pathogen and host data. The development of phylodynamic inference methods relies on models that reconcile evolutionary, epidemiological and immunological dynamics. This then allows the quantification of fundamental parameters using genomic sequence data. This symposium aims to bring together scientists working on both theoretical and experimental aspects of phylodynamics. We envision a mix of people working on computational method development, experimental host-pathogen systems and empirical field studies to exchange the latest developments in the field of phylodynamics. Such a symposium will foster the identification of key future challenges to improve our understanding of the spread of pathogens in host populations.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Gabriel Leventhal and Katja Koelle</p> <p></div></div></p> <p><h3 class="wp-tab-title">13. Evolutionary analysis of ecological communities. Brent Emerson, Andres Baselga</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolutionary analysis of ecological communities</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The application of evolutionary theory and methods for the analysis of communities of species is a developing area of investigation. Evolutionary approaches can facilitate our understanding of the processes governing ecological communities and their spatial variation, against the null hypothesis that they emerge from purely neutral processes. For example, the application of molecular genetic markers within ecological communities potentially provides for (1) an escape from timescale being polarised as either ecological or evolutionary, (2) the integration of dispersal history or interaction effects in community ecology, and (3) a neutral benchmark (i.e. spatial patterns of neutral genetic markers) against which to compare species-level patterns potentially driven by non-neutral processes.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Our proposed symposium is expected to attract broad interest from both empirical and theoretical biologists. The evolutionary analysis of biological communities spans the fields of community genetics, population genetics, phylogeography, community phylogenetics, experimental evolution, macroecology and macroevolution. We are proposing our symposium for three reasons: the area of investigation is (1) emerging, (2) cross-disciplinary, and (3) addresses questions of broad relevance to the ESEB community &#8211; e.g. how do communities of species assemble, interact, and evolve over space and time.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Mike Hickerson and Catherine Graham</p> <p></div></div></p> <p><h3 class="wp-tab-title">14. Experimental evolution and ecology of (microbial and other) ecosystems. Sijmen Schoustra, Susanne Kraemer</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Experimental evolution and ecology of (microbial and other) ecosystems</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">This symposium aims at showcasing recent studies and bringing together ecological and (experimental) evolutionary approaches to study the evolution of ecosystems.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">In natural systems, organisms and species evolve not in isolation but embedded into ecosystems. Several models describe such relationships, for example the Red Queen and niche construction theories as well as metabolism-based models. Such models have shown that evolutionary processes on the ecosystem level can be highly complex. To investigate this phenomenon experimentally, laboratory approaches have simplified the situation by studying evolutionary dynamics using a (very) limited number of strains at a time under controlled laboratory conditions in the powerful approach of experimental evolution. Others studies have utilized a more ecological approach by observing and describing complex systems and how organisms and species can &#8211; or cannot &#8211; co-exist over time.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Increasingly, research has been initiated that combines these two approaches by tracking evolutionary changes of complex ecosystems in laboratory and natural conditions. Examples include studies of the dynamics of species composition in (microbial) ecosystems over space and time and in response to various stresses, the evolution of social interaction between microbes, and long-term co-evolutionary studies between different (sets of) species. For this symposium, we invite submissions of experimental and theoretical studies in this area.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Susanna Remold and Tom Bell</p> <p></div></div></p> <p></div> <p>&nbsp;</p> <h3>Genome evolution</h3> <div id="wp-accordion-4" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">15. Evolution of genomes. Alexandre Reymond, Laurent Keller</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolution of genomes</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">How do genomes change during evolution? What are the mechanisms at play? This symposium will address evolutionary changes ranging from fine-scale genomic changes (e.g. single nucleotide substitution, stop codon gains), to large-scale chromosomal rearrangements (inversion, translocation, fission and fusion) and debate how these are associated with phenotypic traits and speciation. The two invited speakers will present examples picked from the genome of apes. Lucia Carbone will illustrate how lineage specific transposable elements can induce a high frequency of chromosomal rearrangements and how this relate to brachiation, a gibbon specific mode of arboreal locomotion. Evan Eichler will compare the pattern of segmental duplication in primates and other mammals to describe how bursts of segmental duplications precipitated lineage-specific accelerations in large-scale chromosomal rearrangements. We expect that this symposium will attract many speakers and a large audience.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Evan Eichler and Lucia Carbone</p> <p></div></div></p> <p><h3 class="wp-tab-title">16. Evolutionary consequences of selfish genetic elements. Tom Price, Anna Lindholm</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolutionary consequences of selfish genetic elements</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Selfish genetic elements are genes that increase their own transmission at a cost to the rest of the host genome. They are ubiquitous, and come in a wide variety of forms, ranging from transposons, to driving or parasitic chromosomes, to endosymbionts. They have been implicated in the creation of evolutionary novelty, including genomic restructuring, transcriptional changes, the generation of reproductive isolation, and asexuality. Recent work has found they can affect the ecology and survival of populations, resistance against predators, parasites and environmental conditions, and that their selfishness can be condition dependent. This symposium will focus on the evolutionary consequences of selfish genetic elements for their host species, bringing together researchers working on a wide variety of selfish genetic elements and examining their consequences on species, populations, individual phenotypes, and genomes.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">This is timely because</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">a) Despite the similarities between selfish genes, recent meetings and symposia have been on single classes of selfish genes (typically transposons or endosymbionts). A generalist symposium will allow cross-pollination of ideas and techniques, and a broader perspective.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">b) major breakthroughs have recently been made in research on transposons, meiotic drive, and endosymbionts.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">c) the genomics revolution means that in the next few years many new selfish elements are likely to be found and investigated in a wide variety of systems.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Gerry Wilkinson and Laura Ross</p> <p></div></div></p> <p><h3 class="wp-tab-title">17. Polyploid evolution: Integrating ecological and genomic studies. Mario Vallejo-Marin, Richard Buggs</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Polyploid evolution: Integrating ecological and genomic studies</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Polyploidisation is a ubiquitous feature in the evolution of plants and animals, and has been associated with changes in the phenotype and ecology of organisms. We currently have a good understanding of the genomic consequences of polyploidisation, yet, few studies have attempted to bridge ecological and genomic analyses in polyploids. This limitation is due, in part, to the fact that genomic tools were not available for organisms suitable for ecological studies. The rapid development of genomic tools for non-model organisms, have dramatically changed this situation, making possible to work on integrating ecological and genomic studies in polyploid evolution. Our proposal is to organise a symposium, which will bring together researchers studying the ecology and genomics of polyploids, and foster interactions and collaborations between groups working towards the understanding of how genomic changes during polyploidisation affect the evolution of phenotypic and ecological traits in natural populations.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Christian Parisod (U. Neuchatel) works on the interface between genomic and ecological processes in polyploid plants, investigating the origin, and phenotypic consequences of genome re-organisation following polyploidisation in wild species. Andrea Harper (U. York, UK) is an early career scientist who has developed novel methods to study polyploid systems in crop species, and which hold great promise to be applied to natural polyploid species.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Christian Parisod and Andrea Harper</p> <p></div></div></p> <p><h3 class="wp-tab-title">18. How to identify and test the loci and alleles underlying adaptation? Paul Schmidt, Thomas Flatt</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">How to Identify and Test the Loci and Alleles Underlying Adaptation?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">To understand the mechanisms underlying adaptation, causal molecular variants, genes and pathways must be identified, characterized and – ultimately – experimentally verified. To this end, various methods for outlier detection, QTL mapping, and association studies have provided a wealth of “candidates” for phenotypes of interest, the response to artificial and natural selection, and adaptive differentiation within and among taxa. Recent advances in whole-genome sequencing allow an unprecedented, comprehensive evaluation of genotype-phenotype associations. However, one major issue with whole-genome screens is whether any given “candidate” actually represents a true positive: population structure and demography, the number of independent chromosomes, statistical power, and other complications are known to generate false positives. Thus, one of the emerging challenges in evolutionary genomics is to unambiguously identify and empirically validate candidates identified in –omics-level screens. The goal of our symposium is to discuss and showcase how to best identify and validate candidate variants, genes, and pathways. Specifically, the symposium aims to (1) evaluate methods by which candidates are identified and investigated; (2) generate discussion regarding the significance of functional validation of identified candidates in quantitative, ecological and evolutionary genetics; and (3) present some of the best current research related to functional identification and validation.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Felicity Jones and Alistair McGregor</p> <p></div></div></p> <p><h3 class="wp-tab-title">19. Ignoramus et ignorabimus? &#8211; How much genome scans can and should tell us about evolution. Daniel Berner, Marius Roesti</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Ignoramus et Ignorabimus? &#8211; How much genome scans can and should tell us about evolution</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The wave of genome-wide scans for molecular signatures of adaptation is starting to hit genetic non-model organisms. At the same time, such analyses performed in those organisms offering the most powerful genomic resources point to a fundamental issue in evolutionary genomics, already foreshadowed by a century of quantitative genetics: adaptation is highly complex at the genomic level, with many loci involved, most of them probably exhibiting an effect size challenging statistical detection. The goal of this symposium is to bring together contributions from empirical and theoretical genomics to address three main questions: (i) How can our methodological toolkit be optimized to capture the genomic complexity of adaptation? (ii) Where are our limits to understanding and interpreting patterns revealed by genome scans? (iii) How much molecular detail do we need to illuminate for an adequate understanding of adaptive evolution? This symposium thus has a strong methodological, conceptual, and philosophical orientation. Given the enormous amount of financial and human resources currently being directed to genome scans in genetic model and non-model systems – at the expense of more traditional evolutionary investigations – we believe that this symposium will be of outstanding relevance to the development of evolutionary genomics, and to evolutionary biology in general.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Matthew Rockman and Rasmus Nielsen</p> <p></div></div></p> <p><h3 class="wp-tab-title">20. Genomics of local adaptation. Santiago Gonzalez-Martinez, Martin Lascoux</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Genomics of Local Adaptation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The study of how organisms adapt to different environments is one of the major challenges in evolutionary biology. Recent genome and transcriptome sequencing has allowed fast progress in our understanding of the genomic signatures of local adaptation, including the genetic architecture of fitness traits, and the identification of ecologically-relevant gene variation. Genome-wide molecular studies have addressed classical questions on local adaptation, such as the role of new mutation vs. standing variation or the geographical distribution of adaptive polymorphisms. They are also starting to produce the empirical information needed to assess the potential for evolutionary responses of keystone plants and animals in the face of impending climate change, a major societal concern, and to push forward the field from mostly descriptive studies towards the construction of prediction models. Recently, genome-wide studies have also produced detailed knowledge on covariance of allelic effects and adaptive gene networks. Nevertheless, ecological genomic studies of local adaptation also bring about new challenges related to sampling issues, the production of reliable genomic data in many individuals, the analysis of large and particularly complex datasets, and the understanding of the limitations associated with these analyses. This Symposium will welcome original research in model and non-model species as well as the presentation of novel methodological approaches.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Outi Savolainen and Thomas Mitchell-Olds</p> <p></div></div><br/> </div> <p>&nbsp;</p> <h3>Plasticity, epigenetics and behaviour</h3> <div id="wp-accordion-5" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"><br/> <h3 class="wp-tab-title">21. The evolution of phenotypic plasticity within and across generations. Matthew Walsh, Steve Munch</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">The evolution of phenotypic plasticity within and across generations</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">The evolution of phenotypic plasticity has been studied for decades. There is now a substantial body of evidence of demonstrating that the environment can induce phenotypic changes that occur (1) within a generation or (2) non-genetic changes in phenotype that span multiple generations. Yet, these two forms of plasticity are generally evaluated separately. This symposium welcomes research that explores the evolution of plasticity within- or across-generations. The goal is to unite researchers from a diversity of backgrounds and employing current technologies to summarize our current understanding of the selective forces that drive the evolution of plasticity and bridge the gap between research exploring plasticity within or across generations.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Cameron Ghalambor and Eva Jablonka</p> <p></div></div></p> <p><h3 class="wp-tab-title">22. Evolutionary epigenetics: switching from models to the field. Conchita Alonso, Ovidiu Paun</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolutionary Epigenetics: switching from models to the field</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">During the last two decades it has become increasingly clear that epigenetic mechanisms, that do not affect the nucleotide sequence of the genome but regulate gene expression, can cause new phenotypes (epilalleles). Frequently altered by environmental stimuli epigenetic marks can be transmitted from one generation of organisms to the next throughout the germline. Studies focused on model organisms have showed that cytosine DNA methylation is an important epigenetic mark, whose de novo formation and maintenance involves different enzymes, histone modification and RNAs. New developments are expected in the field of molecular epigenetics that will clarify some controversial issues such as the maintenance for multiple generations in absence of the environmental disruption and the potential genetic basis of epigenetic variation.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">However, to unravel the role of epigenetics in adaptation and evolution it is now essential to analyze in a real-world context the magnitude of and environmental effects on epigenetic variance, the correlates with individual fitness, and the inheritance of epigenetic signals (Population Epigenetics), as well as their possible effects on speciation, diversification rate and appearance of evolutionary novelty (Macroevolutionary Epigenetics). This symposium seeks to join evolutionary ecologists, botanists and zoologists, and geneticists working with natural populations to contribute to this end.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Koen Verhoeven and Annalisa Varriale</p> <p></div></div></p> <p><h3 class="wp-tab-title">23. Emerging ‘models’ in evolutionary and ecological neurobiology. Stephen Montgomery, Alison Wright</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Emerging ‘models’ in evolutionary and ecological neurobiology</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Despite their fundamental importance in animal biology and diversification, we know little about the proximate basis of ecologically relevant behaviour in wild populations. Understanding the genetic and neural basis of behavioural evolution is of primary interest, but also reveals mechanistic principles governing the evolution of complex phenotypes. Recent developments in sequencing technology, methods to quantify complex behaviour, and the application of neurological techniques to more diverse organisms, have led to accelerated progress in this field. A number of organisms have attracted particular attention and are emerging as promising ‘model’ systems. Notable insights include the identification of distinct genetic and neural modules underpinning behaviour, the effects of gene expression and patterning on the evolution of brain size/structure, the extent of plasticity or divergence in brain structure between closely related species, and the nature of developmental and functional constraints acting on the brain. This symposium will bring together researchers at the forefront of evolutionary and ecological neurobiology to share methodological developments and identify emerging themes in this rapidly developing field.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Katie Peichel and Niklas Kolm</p> <p></div></div></p> <p><h3 class="wp-tab-title">24. Evolution of behavioral variation. Barbara Feldmeyer, Susanne Foitzik</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Evolution of behavioural variation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">During the last decade, it has become increasingly clear, that individuals of a single species differ consistently in behaviour over time and across different contexts. Recently, researchers started to explore the evolution of this behavioural variation and its fitness consequences, for example in studies on animal personalities or on behavioural castes in social animals. It also becomes evident that standardized lab experiments are valuable to characterize different behavioural types whereas experimental field studies are needed to characterize their performance in an ecological context. Moreover, advances both in statistical methods as well as in genomic tools now allow to disentangle and to identify different sources of behavioural variance. We still know little about behavioural genes and their regulatory mechanisms, how conserved these genes are between species and how behavioural variance is mediated on a genetic level. In this symposium, we plan to bring together scientists from various disciplines to develop a broad picture on the current research on the evolution of behavioural variation and its genetic underpinning.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Seirian Sumner and Jürgen Gadau</p> <p></div></div><br/> </div> <p>&nbsp;</p> <h3>Microbe evolution</h3> <div id="wp-accordion-6" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">25. Groups versus individuals: levels of selection in microbial systems. Christian Kost, Martin Ackermann</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Groups versus individuals: levels of selection in microbial systems</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">This symposium focuses on how different levels of selection shape the link between genotype and phenotype in microbial systems. A number of recent discoveries have revolutionized our understanding of the molecular mechanisms that govern the expression of microbial phenotypes as well as the selective conditions that determine their evolution in the long-run. For example, microbes combine stochastic molecular processes with signals they receive from their environment and from conspecifics to produce phenotypic diversity in clonal populations, and to engage in behaviours that have selective consequences for themselves as well as for the group. Moreover, by combining metabolic functions among microorganisms of the same or different species, new group-level phenotypes can emerge that drastically impact selection acting on individual cells. The aim of this symposium is to discuss how selection at different levels of biological organization affects the expression of phenotypes as well as to analyze how cell-level phenotypes can promote functionality at the level of groups. By drawing together empirical and theoretical contributions from evolutionary and quantitative biology, this interdisciplinary symposium will help to identify future research avenues and facilitate discussions among scientists that otherwise would not meet.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Martin Polz and Thierry Emonet</p> <p></div></div></p> <p><h3 class="wp-tab-title">26. Real-time bacterial evolution in vivo and in vitro. Daniel Wilson, Craig MacLean</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Real-time bacterial evolution in vivo and in vitro</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Bacterial microevolution is important in determining host-pathogen interactions, informing microbial forensics, and characterizing the emerging threat of antibiotic resistance. Genomics is affording unprecedented insights into these problems by casting light on the adaptive and non-adaptive processes driving short-term evolution of bacterial populations. The aim of this symposium is to synthesize the recent profusion of experimental and observational work in real-time bacterial evolution, and to prompt a synergistic approach to understanding the selective forces driving bacterial evolution in the laboratory and within the host.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Sebastien Gagneux and Marvig Rasmus Lykke</p> <p></div></div></p> <p></div> <p>&nbsp;</p> <h3>Selection / adaptation</h3> <div id="wp-accordion-7" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">27. Ecology and evolution of floral signals. Florian Schiestl, Martin von Arx</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Ecology and evolution of floral signals</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Much of the striking diversity in angiosperms is comprised by floral signals, i.e. color, fragrance and shape of flowers. Floral displays are multimodal, complex communication channels of plants, shaped by selection mediated through mutualists (pollinators) and antagonists (e.g. herbivores). Floral signals can have important consequences for plant diversification, as they can act as filters, mediating assortative pollinator visitation and thus floral isolation among co-occurring plant species. Adaptive population divergence in floral signals is common, and can be the first step to speciation or reproductive character displacement. In the past decade, we have seen a great increase of experimental studies, manipulating floral signals and measuring related fitness consequences. Also, floral signals are more and more investigated in the context of a plants&#8217; signaling dilemma, by attracting mutualists while remaining invisible or deterring antagonists. Floral signals have been studied in the context of phenotypic selection, reproductive isolation, sensory preferences and pre-existing bias of flower visitors, and convergence within pollination systems. In our proposed symposium we seek to unify classical approaches using pollinators as selective agents with more recent focus on anatgonists and their role in floral signal evolution. We also strive to include approaches that combine multiple sensory modalities.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Brian Smith and Santiago Benitez-Vieyra</p> <p></div></div></p> <p><h3 class="wp-tab-title">28. Variation in natural selection: patterns, causes, and consequences. Anne Charmantier, Michael Morrissey</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Variation in natural selection: patterns, causes, and consequences</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Despite longstanding interest in the dynamics of selection in space and time and its impact on adaptive evolution, the topic has been remarkably resistant to general empirical progress. This symposium will gather contributions that address the following fundamental questions using theoretical, empirical and experimental approaches:</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">• How much does natural/sexual selection vary in time and space?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">• What is the evolutionary importance of fluctuating selection? In particular, can variation in selection explain evolutionary stasis, that is, the absence of microevolution in traits that are heritable and appear under strong directional selection?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">• What are causes (ecological drivers) of variation in selection?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">• Under what conditions does spatial variation in selection lead to local adaptation?</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">These long-standing questions may remain unresolved in part because of a lack of a conceptual framework to guide empirical studies. Recent theoretical and synthetic studies have begun to provide such a framework. We will bring together theoreticians and empiricists, and people who have recently been developing methods specific to understanding variation in selection. The symposium with therefore be particularly timely in moving this topic forward.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Christina Caruso and Luis-Miguel Chevin</p> <p></div></div></p> <p><h3 class="wp-tab-title">29. The evolution and ecology of trait loss and dependency. Jacintha Ellers, Fabrice Vavre</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">The evolution and ecology of trait loss and dependency</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Loss of traits can dramatically alter the fate of species, but the evolutionary importance of trait loss has long been undervalued. Trait loss or decay can occur when a change of environment alters selection pressure making reduced trait expression neutral or beneficial. Recent work has expanded our view on the evolutionary trajectories leading towards trait loss, broadening the spectrum to manipulation, conflict and production of public goods as drivers of trait loss and emphasizing the role of natural selection. But it has remained a challenge to understand the molecular mechanisms of trait loss. What is the genetic basis of trait loss? Is trait loss at the phenotypic level always followed by gene loss? Or does it involve changes in gene expression or a rewiring of the gene network? Another lack of knowledge concerns the ecological consequences of trait loss, especially when trait loss takes place as a result of species interactions. Evolved dependencies through trait loss may either reinforce cooperation between partners or increase susceptibility of the symbiosis to environmental variations.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">In this symposium we want to discuss recent advances in trait loss research, focusing on the conditions leading to trait loss, the genetic and genomic basis of trait loss, and the ecological and community effects of dependency through trait loss.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Sonia Pascoal and Amparo Latorre</p> <p></div></div></p> <p><h3 class="wp-tab-title">30. Protein evolution: structural and functional perspective. Romain Studer, Maria Anisimova</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Protein evolution: structural and functional perspective</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Proteins evolve by the replacement of amino acids (substitutions) or the insertion/deletion of fragments (indels). For the protein, mutations may be deleterious or beneficial, governed by the laws of natural selection. Beneficial mutations increase the fitness of the phenotype and are more likely to become fixed in the genome (positive selection). Proteins are not robust to drastic changes (i.e. important changes in stability) and mutations that favour an adaptive functional change are generally accompanied by other coevolving mutations that insure the integrity of the 3D structure (compensatory effect). All these biophysical properties are paving the way for protein evolution. Traditionally, there was little if any crosstalk between the fields of protein biophysics, protein structure-function and molecular evolution. The last several years have seen some exciting development in merging these areas to obtain an in-depth understanding of how proteins evolve. For example, a better understanding of how structural constraints affect protein evolution will greatly help to optimise stochastic models of sequence evolution. The symposium aims at exploring this new synthesis.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Dan S. Tawfik and Richard A. Goldstein</p> <p></div></div></p> <p><h3 class="wp-tab-title">31. Melanism: macrophysiology to molecules. Subhash Rajpurohit, Paul Schmidt</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Melanism: macrophysiology to molecules</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">In nature, organisms display extreme variation for multiple aspects of pigmentation phenotype; this is commonly observed both within and among populations, as well as among taxa. Among populations across environmental gradients (at the macro-physiological level), pigmentation has been found to be associated with several fitness components such as mating ability, disease resistance, UV tolerance, thermal adaptation, and drought tolerance. From the molecular perspective, a number of genes and causal genetic changes underlying pigmentation phenotype have been identified in multiple taxa. The established connections between molecular variation, physiology, and fitness make this trait ideally suited for addressing fundamental questions at the crossroads of ecology, evolution and physiology.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Comprehensively defining the connections between physiological processes, pathways and causative molecules is a major challenge in evolutionary physiology. The melanin formation pathway is a multistep and complicated process and its physiological nature is poorly understood. We take advantage of this symposium to address questions related to melanism and its adaptive nature.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Note: We have received interest from a number of other investigators including women scientists. Our approach also covers representation of different taxa and geographical regions.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Alexandre Roulin and Aya Takahashi</p> <p></div></div></p> <p><h3 class="wp-tab-title">32. Forecasting eco-evolutionary responses to global changes. Frédéric Guillaume, Ophélie Ronce</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Forecasting eco-evolutionary responses to global changes</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Evolutionary biology is seldom seen as a predictive science mostly because evolutionary changes are traditionally expected to occur over long time scales where it becomes impossible to predict evolutionary trajectories. Current evidence for rapid adaptive changes on short time scales challenges this vision and argues for the inclusion of evolutionary responses into ecological niche modelling of shifting species&#8217; distributions under climate changes. Ecological forecasting of future species&#8217; ranges has been preferred based on the premise of conservatism of species&#8217; fundamental ecological niches on the time scale of global change. The question of niche conservatism is currently intensely debated and a role for evolutionary adaptation in niche dynamics is expected. Current niche modelling predictions of species extinctions may thus be inaccurate whenever species have the capacity to adapt to novel conditions outside their niche. On the other hand, evolutionary processes may aggravate the consequences of environmental changes on species persistence whenever the evolution of adaptive traits is limited by genetic or demographic constraints. This symposium aims at highlighting recent efforts to bring together ecological and evolutionary approaches to better understand and predict the potential responses of natural species to environmental changes and the impact of ongoing global changes on the maintenance of biodiversity.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Katja Schiffers and Liana Burghardt</p> <p></div></div><br/> </div> <p>&nbsp;</p> <h3>Speciation</h3> <div id="wp-accordion-8" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">33. The molecular basis of adaptation and ecological speciation. Philipp Schlüter, Shuqing Xu</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">The molecular basis of adaptation and ecological speciation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">We aim to bring together people who 1) study the molecular basis of adaptations to the biotic (e.g. pollinators, herbivores, parasites) or abiotic (e.g. soil, habitat) environment and 2) who study how divergent selection drives species divergence on the ecological and genomic levels.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Research in adaptation and speciation is often divided into studies of molecular mechanisms of adaptations, and study of the ecological drivers and genomic patterns of speciation. However, integrating both approaches can provide complementary understanding. Therefore, bringing people from these fields together will promote the exchange of ideas; knowledge of different angles and methodologies used will create a synergistic effect on adaptation and speciation research.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Although the joint study of molecular adaptation and speciation is of general evolutionary interest, research on plants in this area has been underrepresented. We therefore wish to specifically emphasise progress in plant systems and to encourage such contributions. This will allow us to better compare and contrast findings from different groups of organisms, and thus to broaden research in the fields of adaptation and speciation.</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Mark Rausher and Beverley Glover</p> <p></div></div></p> <p><h3 class="wp-tab-title">34. Charting the genomic landscape of speciation. Anja Marie Westram, Mark Ravinet</h3><br/> <div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Charting the genomic landscape of speciation</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Understanding how speciation with gene flow works at the genomic level is currently a major focus of speciation biology. Genome scans between diverging populations have become widespread, demonstrating that the extent of differentiation can be quite variable across the genome. Highly differentiated regions are often interpreted as resulting from divergent selection, therefore playing an important role in speciation. This view is prominently expressed in the striking metaphor of ‘speciation islands’. Yet some debate remains. Is differentiation generated by divergent natural selection acting as a barrier to otherwise homogenising gene flow? Or by recombination rate variation and/or intrinsic incompatibilities? Might differentiation occur when gene flow ceases and local adaptation increases the rate of lineage sorting in some parts of the genome but not others? And what genomic features affect homogenising gene flow? Being able to distinguish these alternative explanations is fundamental for understanding the genomic basis of speciation with gene flow. This symposium will explore this debate, bringing together empirical evidence from different perspectives and focusing on emerging approaches for identifying the processes that cause the rugged landscape of genomic differentiation. (The idea for the symposium was conceived by a group of four people who are all contributing to its organisation: Anja Westram, Sheffield, UK; Mark Ravinet, NIG, Japan; Juan Galindo, Vigo, Spain; Rui Faria, Porto, Portugal)</p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 12pt;color: #666">Invited speakers: Nicolas Bierne and Mohamed Noor</p> <p></div></div></p> <p></div> <p>&nbsp;</p> <h3>Open symposium</h3> <div id="wp-accordion-9" class="wp-accordion ui-lightness jqui-styles wpui-tabs-vertical wpui-collapsible" data-style="ui-lightness"></p> <p><h3 class="wp-tab-title">35. Open symposium. John Pannell, Laurent Keller</h3></p> <p><div class="wp-tab-content"><div class="wp-tab-content-wrapper"></p> <p class="police" style="font-family: helvetica, sans-serif;font-size: 14pt;color: #0093d5">Open symposium</p>
